### PR TITLE
Gridcoin Whitepaper 1.0.1

### PR DESCRIPTION
**Pull request by wisecracka**
Reentered the hyperlinks for all links of page 12 of the Whitepaper.

### CODE REVIEW

File: white_paper_source/Document fonts/AdobeFnt19.lst [hunk 1]
```diff
@@ -11,14 +11,14 @@ FullName:Minion Pro
 MenuName:Minion Pro
 StyleBits:0
 WritingScript:Roman
-OutlineFileName:F:\Design Work\Creative Cloud Files\Gridcoin Graphics\Whitepaper\Gridcoin_White_Paper_1.0.0 Folder\Document fonts\MinionPro-Regular.otf
+OutlineFileName:C:\Users\Bill\GitHubLocalRepos\Whitepaper\white_paper_source\Document fonts\MinionPro-Regular.otf
```
Comment (tomasbrod) [inline white_paper_source/Document fonts/AdobeFnt19.lst:14]: File contains reference to local file system. Probably bogous.

**Review by tomasbrod — COMMENTED** [review]
* I Can't review changes to binary files.
* Whole file (Gridcoin_White_Paper_1.0.1.md) was added. Seems like you wanted to replace `Gridcoin_White_Paper_1.0.0.md`. If so, then remove the old version.

I do not know If I should be reviewing this pr, but I seem to have rights to merge it. I wont merge it, because I can't really review it properly. Hopefully someone more competent will give review.

File: white_paper_source/Document fonts/AdobeFnt19.lst (hunk 1 above)
Comment (wisecracka) [reply white_paper_source/Document fonts/AdobeFnt19.lst:14]: The AdobeFnt19.lst file is created when creating a package in InDesign. It lets InDesign know what fonts are being used in the document. I'd assume the reference to my file system is so InDesign can know where the fonts are.

**Comment by wisecracka** [comment]
Good catch on `Gridcoin_White_Paper_1.0.0.md`, I wanted to add it to the archive and screwed it up.

I'm not sure how you can review the indd and imdl files without InDesign. The changes to the pdf were corrections to links on page 12.

**Comment by wisecracka** [comment]
`Gridcoin_White_Paper_1.0.0.md` has had the changes reverted. Should just be a move to the archive section now.

**Comment by barton2526** [comment]
Looks good to me 👍

**Pull request merged**